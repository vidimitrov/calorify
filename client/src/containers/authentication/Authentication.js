import React from 'react';
import PropTypes from 'prop-types';

export const Authentication = ({ children }) => (
  <div>{children}</div>
);

Authentication.propTypes = {
  children: PropTypes.oneOf([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node,
  ]).isRequired,
};

export default Authentication;
