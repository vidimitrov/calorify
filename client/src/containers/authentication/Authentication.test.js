import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Authentication from './Authentication';

it('Authentication container renders correctly', () => {
  const tree = renderer.create(
    <Authentication><div /></Authentication>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
