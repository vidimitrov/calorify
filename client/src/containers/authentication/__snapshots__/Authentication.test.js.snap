// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication container renders correctly 1`] = `
<div>
  <div />
</div>
`;
